server/src/util/validateEnv.ts

```diff
@@ -0,0 +1,21 @@
+import { cleanEnv } from 'envalid';
+import { port, str } from 'envalid/dist/validators';
```

Comment:
ive not used this package so I will have to look up what it does but cool

Reply:
yer i also dont really have much knowledge but it seems to work for now